src/templates/layouts/vetCenterOutstation/vetCenterOutstation.stories.ts

```diff
@@ -0,0 +1,205 @@
+import { Meta, StoryObj } from '@storybook/react'
+import { VetCenterOutstation } from './index'
+import { Wysiwyg as FormattedWysiwg } from '@/types/formatted/wysiwyg'
+import { QaSection as FormattedQaSection } from '@/types/formatted/qaSection'
+
+const meta: Meta<typeof VetCenterOutstation> = {
+  title: 'Layouts/VetCenterOutstation',
+  component: VetCenterOutstation,
+}
+export default meta
+
+type Story = StoryObj<typeof VetCenterOutstation>
+
+export const Example: Story = {
+  args: {
+    address: {
```

Comment:
Non-blocking question: Is there any way to leverage the mock data, or do we need to re-define everything here too because of Reasons™?

Reply:
I think reusing mocked formatted data is the best. If it's possible.